src/gei/Program.cs

```diff
@@ -36,6 +37,7 @@ public static async Task Main(string[] args)
                 .AddSingleton<IVersionProvider, VersionChecker>(sp => sp.GetRequiredService<VersionChecker>())
                 .AddTransient<ITargetGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
                 .AddTransient<ISourceGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
+                .AddTransient<ISecretScanningAlertServiceFactory>(sp => sp.GetRequiredService<GitHubSecretScanningAlertServiceFactory>())
```

Comment:
There is no need to resolve the `GitHubSecretScanningAlertServiceFactory` here, so the registration can be simplified:
```suggestion
                .AddTransient<ISecretScanningAlertServiceFactory, GitHubSecretScanningAlertServiceFactory>()
```

The reason that `ITargetGithubApiFactory` is registered like that is to resolved the same `GithubApiFactory`. (we are going to do a refactor later on to register everything as a singleton, which will simplify the registrations)